app/assets/stylesheets/application.tailwind.css

```
@@ -38,8 +38,37 @@
   }
 }
 
+/* 自分の名前をつけた変数にスタイルを当てる */
 @layer components { 
-  .btn, .link {
+  /* ボタン・ボタン風リンクなどの押下できるcardに適用 */
+  .card {
     @apply cursor-pointer shadow-md shadow-black/40 hover:-translate-y-0.5 hover:shadow-lg active:translate-y-1 active:shadow-sm transition-all;
   }
+  
+  /* 編集・削除を表示させるスワイプ用のcardの外側に適用*/
```

Stylelint (17.6.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)